docs/src/tut/community-insurance/index.md

```diff
@@ -0,0 +1,213 @@
+* Initialize a react app (using [create-react-app](https://tailwindcss.com/docs/guides/create-react-app)) and configure tailwind css.
+
+## Initialize the application
+Go to the directory where you wish to create your project and open a new terminal from there.
+
+Run these commands to initialise the project
+```cmd
+$ npx create-react-app community-insurance-dapp
+$ cd community-insurance-dapp
+```
+
+To Install Tailwind CSS,
+```cmd
+$ npm install -D tailwindcss postcss autoprefixer
+$ npx tailwindcss init -p
+```
+
+To Configure Tailwind CSS, open the file called `tailwind.config.js` at the root and paste in the following code.
+```
+load: /examples/insurance/tailwind.config.js
+```
+Open your `src/index.css` file and paste the code below, overwriting everything inside.
+```
+load: /examples/insurance/src/index.css
+```
+
+At the root of your application code, there a `package.json` file which contains all the configuration for react 
+and it is responsible for keeping track of all the dependances required to run this application. 
+Open it and paste the code bellow, overwriting everything inside.
+
+```
+load: /examples/insurance/package.json
+```
+
+Now run the command bellow and wait for npm to finish installing all React dependances
+```cmd
+$ npm install
+```
+You have now finished preparing your react frontend app. What about the Reach backend ? 
+Well, if you had successfully completed [the basic tutorial](##tut), by now you know that at the root of 
+the reach project there is a file named `index.rsh`, 
+as well as the reach script file which allows you to run the command `./reach xxx`. 
+To integrate both reach and react, all you need is to make sure that the output file generated after compiling the `.rsh` 
+file resides inside the react project so that it can be imported at the top of react code. 
+To achieve this, create a folder inside the src/ folder of the react app where the reach compilation output will go. 
+While you can name it whatever you want, I suggest `reach-build`. So your reach compiled output should be place inside 
+`community-insurance-dapp/src/reach-build`. By the way, you can direct the output to go there at the time of running your 
+commpillation command. By default `./reach compile` command places the output in a `build` folder at the root of the reach app.
+You can change the destination of the output by specifying the `-o` flag like so: 
+```cmd
+$ REACH_CONNECTOR_MODE="ALGO-browser" ./reach compile -o community-insurance-dapp/src/reach-build
+```
+The above command assumes that your `index.rsh` file is in the same directory with the `community-insurance-dapp` folder.
+If that's not your case, you will have to adjest the command accordingly.
+
+Preparation is done. Now lets start writing our Reach application code.
+
+## Create your Reach program file for the community insurance dapp
+```
+load: /examples/insurance/index.rsh
+range: range: 1-60
```

Comment:
It seems that the codebase has changed since the guide was drafted. Think you mean to load lines 1-54.
However, I'd suggest breaking it up more. Load `Insurer` and talk about their methods, then load `CommunityMember` and work through their code. 

A more detailed tutorial would have directories in differing stages of development and build the codebase over several steps, (as in the RPS tutorial) but you can introduce this as a guide if you'd prefer to introduce the entire codebase as a complete entity.